compute/cloud-client/instances/test/instancesTest.php

```diff
@@ -170,4 +169,25 @@ public function testSetUsageExportBucketCustomPrefix()
         ]);
         $this->assertStringContainsString('project `' . self::$projectId . '` is disabled', $output);
     }
+    public function testListAllImages()
```

Comment:
We want new lines between functions

```suggestion

    public function testListAllImages()
```